### ISSUE TITLE
x/tools/gopls: semantic tokens classifies "return" and "false" as the same

### ISSUE THREAD

**Issue by stamblerre** [golang/go#42682]
`return false` has the same coloring for both words--the default textmate syntax highlighting has different colors for the two. I'm guessing that `false` is being classified as a keyword rather than a value?

<img width="489" alt="Screen Shot 2020-11-17 at 5 14 55 PM" src="https://user-images.githubusercontent.com/5856771/99457129-797a7100-28f8-11eb-8a45-381cc096403e.png">

/cc @pjweinb

**Comment by pjweinb** [comment]
yes. false and true are classified as keywords when they are used as
pre-defined boolean constants. Typescript (and therefore semantic tokens)
doesn't have any way to describe that. (iota is treated the same, for lack
of anything better to do with it, although maybe it should be a readonly
variable)

semantic tokens handles various weird cases better than the textmate grammar
var double = 4
type foo struct {
iota: bool
}
const (
f int = iota
g
iota int = 4
)

**Comment by pjweinb** [comment]
Obsolete
